### DISCUSSION TITLE
Errors when trying to install widgets

### DISCUSSION THREAD

**Discussion by 4164696f73** [streetturtle/awesome-wm-widgets#325] [General]
I'm following the readme, I download the repo, I put the folder in .config/awesome/volime-widget, add "local volume_widget = require('awesome-wm-widgets.volume-widget.volume')" under "require("awful.hotkeys_popup.keys")" and put "volume_widget()," right after "wibox.widget.systray(),", and this is what I get: https://i.imgur.com/dPwa4cs.png

Could anyone help me? I'm totally new to Awesome, literally just started, and all I wanna do is to control my volume with keyboard instead of having to use my mouse.

**Comment by 4164696f73** [comment]
Edit: Chose to go without any widgets. I used:

    awful.key({          }, "XF86AudioMute", function () os.execute("pactl set-sink-mute 0 toggle") end,
              {description = "mute volume", group = "launcher"}),

    awful.key({          }, "XF86AudioRaiseVolume", function () os.execute("pactl -- set-sink-volume 0 +1%") end,
              {description = "mute volume", group = "launcher"}),

    awful.key({          }, "XF86AudioLowerVolume", function () os.execute("pactl -- set-sink-volume 0 -1%") end,
              {description = "mute volume", group = "launcher"}),

instead. Sorry for spam.

**Comment by RadinShirzad** [comment]
> I'm following the readme, I download the repo, I put the folder in .config/awesome/volime-widget, add "local volume_widget = require('awesome-wm-widgets.volume-widget.volume')" under "require("awful.hotkeys_popup.keys")" and put "volume_widget()," right after "wibox.widget.systray(),", and this is what I get: https://i.imgur.com/dPwa4cs.png
> 
> Could anyone help me? I'm totally new to Awesome, literally just started, and all I wanna do is to control my volume with keyboard instead of having to use my mouse.

I have the exact same error here and I am sure I'm doing something wrong any kind of help would be greatly appreciated.
